views/supporters.erb

```diff
@@ -18,6 +18,12 @@
         <h1 class="content-heading text-center">Podporovatelia</h1>
         <div class="content-wrap logos">
           <p>Slovensko.Digital je komunita. Ďakujeme všetkým, ktorí sa rozhodli nás podporiť.</p>
+          <div class="row">
+            <div class="col-sm-3"><a href="/podporovatelia/2019">Podporovatelia v roku 2019</a></div>
+            <div class="col-sm-3"><a href="/podporovatelia/2018">Podporovatelia v roku 2018</a></div>
+            <div class="col-sm-3"><a href="/podporovatelia/2017">Podporovatelia v roku 2017</a></div>
+            <div class="col-sm-3"><a href="/podporovatelia/2016">Podporovatelia v roku 2016</a></div>
```

Comment:
Toto nie je uplne stastne riesenie. Tie prepinatka vyzeraju nefunkcne, lebo loga sa nemenia, len zoznamy pod nimi. Pridal by som ten rok aspon do nadpisu, nech je vidno zmenu. 

Pod napis by som potom mozno hodil tie linky dakto takto:
![image](https://user-images.githubusercontent.com/1374745/60346040-a0aa6880-99ba-11e9-913b-1f6a45a0592c.png)


A zoznam firiem je up-to-date, alebo zatial cakas na data?

Reply:
Este by som navrhol namiesto `/podporovatelia/2019` spravit  `/podporovatelia` (a dotiahnut tam podporovatelov za r. 2019). Podstranky tym padom budu len na minule roky. cc @jsuchal

Reply:
Cakam na data, zoznam firiem je vsade rovnaky len ako placeholder.

Reply:
Malo by to byt. Az na tie firmy

Reply:
@michalrohacek256 skusis nieco vytiahnut z vyrocnych sprav? https://slovensko.digital/vyrocne-spravy

Reply:
pozriem sa na to

Reply:
presiel som tie vyrocne spravy - su v nich len loga firiem (ako na stranke, grid bez uvedenia prispevkov) a nizsie nejake sumare, ale menovite firmy s jednotlivymi sumami som nenasiel